domains/dnaorcode.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Danielnaor",
+        "email": "dnaorcode@gmail.com"
+    },
+    "record": {
+        "TXT": "192.168.68.56"
```

Comment:
Invalid TXT record. No other records provided with TXT. Did you mean to provide an A record? If so, please fix.

Reply:
The IP is also a local IP.